src/AutoRest.CSharp/Common/AutoRest/Plugins/Configuration.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace AutoRest.CSharp.AutoRest.Plugins
 {
-    internal class Configuration
+    internal static class Configuration
```

Comment:
We probably need a different namespace here

Reply:
suggestion?